### PR TITLE
Release/v21.03 slash

### PR DESCRIPTION
**Pull request by kaushik691**
Modified code w.r.t 21.03-slash

### CODE REVIEW

File: .github/workflows/nightly.yml [hunk 1]
```diff
@@ -54,20 +49,10 @@ jobs:
       # Navigating to the directory and installing the requiremenets file
       - name: Install requirements for Dgraph and Slash
         run: |
-          hostname
-          hostname -I
-          sudo chmod -R 777 ${{ github.workspace }}
```
Comment (souravDgraph) [inline .github/workflows/nightly.yml:59]: I think it is better we  keep it, to avoid any perm issues.

File: .github/workflows/nightly.yml [hunk 2]
```diff
@@ -86,17 +71,11 @@ jobs:
...
          
       - name: Run Robot Framework Testcases 01
-        continue-on-error: true
         run: |
           echo ${{ github.workspace }}
           cd ${{ github.workspace }}/Automation_workspace
-          ps -ef | grep dgraph | grep -v grep | awk '{print $2}' | sudo xargs -r kill -9
-          
           # ---- Preparing 21million dataset. ----
           sed -i "s/1million/21million/g" test_suites/dgraph/Linux/dgraph_suite_01.robot
```
Comment (souravDgraph) [inline .github/workflows/nightly.yml:80]: Are these needed anymore? We can pass it as arg in the robot command using -v

File: .github/workflows/nightly.yml [hunk 3]
```diff
@@ -106,35 +85,25 @@ jobs:
...
-          # These are commented as we do not have a fixed bearer token, once we have those we will enable the below TCs.
-          # echo " ---- Running Slash Testcases ----"
-          # robot test_suites/slash/slash_api/sanity.robot
       
-      - name: Run Robot Framework Testcases 03
-        run: |
-          echo ${{ github.workspace }}
-          cd ${{ github.workspace }}/Automation_workspace
-          ps -ef | grep dgraph | grep -v grep | awk '{print $2}' | sudo xargs -r kill -9
+#       - name: Run Robot Framework Testcases 04
```
Comment (souravDgraph) [inline .github/workflows/nightly.yml:97]: Clear these comments.

File: test_suites/dgraph/Linux/dgraph_suite_04.robot [hunk 4]
```diff
@@ -18,45 +17,29 @@ TC_01 Perform live load data.
...
-     Create NFS Backup      1
-     log        ${LATEST_VERSION_CHECK}
-     Run Keyword If     ${LATEST_VERSION_CHECK}     Perform a restore on backup latest versions    0
-     ...    ELSE    Perform a restore on backup by older dgraph versions
-     Clear Backup Folders   ${TRUE}
+     Execute Parallel Loader with rdf and schema parameters    ${rdf_file}    ${schema_file}    is_learner=${TRUE}
 
-TC_04 Perform parallel live and bulk load on data
+TC_03 Perform parallel live loads.
      [Documentation]    Perform live load data.
```
Comment (souravDgraph) [inline test_suites/dgraph/Linux/dgraph_suite_04.robot:28]: parallel

**Review by souravDgraph — APPROVED** [review]

File: .github/workflows/nightly.yml (hunk 1 above)
Comment (kaushik691) [reply .github/workflows/nightly.yml:59]: right now I'm trying only with github containers

Comment (kaushik691) [reply .github/workflows/nightly.yml:59]: will check

File: .github/workflows/nightly.yml (hunk 2 above)
Comment (kaushik691) [reply .github/workflows/nightly.yml:80]: will change

File: .github/workflows/nightly.yml (hunk 3 above)
Comment (kaushik691) [reply .github/workflows/nightly.yml:97]: okay

File: test_suites/dgraph/Linux/dgraph_suite_04.robot (hunk 4 above)
Comment (kaushik691) [reply test_suites/dgraph/Linux/dgraph_suite_04.robot:28]: okay

**Pull request merged**